routes/videos.js

```diff
@@ -46,9 +47,10 @@ router.post('/', function (req, res) {
     }
 
     var video = new Videocall()
-    video.name = req.body.name
-    video.sessionId = session.sessionId
-    video.tokenId = session.generateToken()
+    randName = videoServices.generateChatName(function (name) {
+      video.name = name
+      video.sessionId = session.sessionId
+      video.tokenId = session.generateToken()
 
     video.save(function (err) {
```

Comment:
I think we're missing an ending semicolon for the overall function after i merged the changes from master into this
Can you do a git pull and make sure it compiles/runs